wgengine/magicsock/magicsock.go

```diff
@@ -1318,7 +1318,14 @@ func (c *Conn) sendDiscoMessage(dst netip.AddrPort, dstKey key.NodePublic, dstDi
 		// Can't send. (e.g. no IPv6 locally)
 	} else {
 		if !c.networkDown() {
-			c.logf("magicsock: disco: failed to send %v to %v: %v", disco.MessageSummary(m), dst, err)
+			if isPMTUError(err) {
+				metricSentDiscoPeerMTUErrors.Add(1)
+				if debugPMTUD() {
+					c.logf("magicsock: disco: failed to send to %v, PMTU error %v", dst, err)
```

Comment:
is this log sufficient? @sailorfrag @jwhited

Reply:
https://github.com/tailscale/tailscale/pull/10870/files#r1456400869